susemanager-utils/susemanager-sls/salt-ssh/preflight.sh

```diff
@@ -247,14 +247,14 @@ if [ -n "${VENV_SOURCE}" ]; then
             mv usr usr.tmp
             mv usr.tmp/lib/venv-salt-minion/* .
             rm -rf usr.tmp
+            if [ ! -x bin/python ]; then
+                rm -f "${VENV_TMP_DIR}/${VENV_HASH_FILE}"
+                exit_with_message_code "Error: Unable to extract the bundle from ${TEMP_DIR}/${VENV_PKG_FILE}!" 6
+            fi
```

Comment:
Is moving the place correct? In my case it would mean that x bit of bin/python is not checked at all.

Reply:
yes, as it make sense only for extracting the package